src/pages/expire-password/index.page.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { memo, useCallback, useState } from 'react';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+
+import { path } from 'pages/routes';
+
+import { resendEmail } from 'resources/user/user.api';
+
+import Button from 'components/Button';
+import useHandleError from 'hooks/useHandleError';
+
+import styles from './styles.module.css';
+
+const ForgotPassword = () => {
+  const router = useRouter();
+  const handleError = useHandleError();
+
+  const { email } = router.query;
+
+  const [loading, setLoading] = useState(false);
+  const [isSent, setSent] = useState(false);
+
+  const onSubmit = useCallback(async () => {
+    try {
+      setLoading(true);
+
+      await resendEmail({ email });
+
+      setSent(true);
+    } catch (e) {
+      handleError(e);
+    } finally {
+      setLoading(false);
+    }
+  }, [email, handleError]);
+
+  if (isSent) {
+    return (
+      <div className={styles.container}>
```

Comment:
we don't have <Head> in this place.

when we will sent an email, is the page title will be Password reset link expired?